src/navigation/menu_bar.py

```diff
@@ -200,17 +183,17 @@ def _save_file_at_path(self, path: str, text: str) -> None:
 
     def _show_scroll(self, title: str, text: str) -> None:
         """Shows a MessageDialog with a certain title and text"""
-
-        async def coroutine(self: ThoughtBox) -> None:
+        # Type annotation is ThoughtBox but can't due to partial initialization
+        async def coroutine(self: object) -> None:
```

Comment:
Isn't the type `MenuNav` since that is what self is?

Reply:
You're right.